src/main/java/org/kainos/ea/team2/db/DatabaseConnector.java

```diff
@@ -0,0 +1,38 @@
+package org.kainos.ea.team2.db;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DatabaseConnector {
+    private static Connection conn;
+
+    public static Connection getConnection() throws SQLException {
+        String user,password,host, name;
+
+        if(conn != null && !conn.isClosed()) { return conn; }
+
+        try(FileInputStream propsStream = new FileInputStream("db.properties")) {
+            Properties props = new Properties();
+            props.load(propsStream);
+
+            user = props.getProperty("user");
+            password = props.getProperty("password");
+            host = props.getProperty("host");
+            name = props.getProperty("name");
```

Comment:
could you load properties from environment variables rather than db.properties (have a look at unit testing repo from last week for reference)

Reply:
Changed to reflect this.